feature/author/src/main/java/com/google/samples/apps/nowinandroid/feature/author/AuthorViewModel.kt

```diff
@@ -54,75 +56,67 @@ class AuthorViewModel @Inject constructor(
         authorId = authorArgs.authorId,
         userDataRepository = userDataRepository,
         authorsRepository = authorsRepository
-    )
-        .stateIn(
-            scope = viewModelScope,
-            started = SharingStarted.WhileSubscribed(5_000),
-            initialValue = AuthorUiState.Loading
-        )
+    ).stateInViewModelScope(viewModelScope, initialValue = AuthorUiState.Loading)
 
-    val newsUiState: StateFlow<NewsUiState> =
-        getSaveableNewsResourcesStream.newsUiStateStream(authorId = authorArgs.authorId)
-            .stateIn(
-                scope = viewModelScope,
-                started = SharingStarted.WhileSubscribed(5_000),
-                initialValue = NewsUiState.Loading
-            )
+    val newsUiState: StateFlow<NewsUiState> = getSaveableNewsResourcesStream
+        .newsUiStateStream(authorId = authorArgs.authorId)
+        .stateInViewModelScope(viewModelScope, initialValue = NewsUiState.Loading)
 
-    fun followAuthorToggle(followed: Boolean) {
-        viewModelScope.launch {
-            userDataRepository.toggleFollowedAuthorId(authorArgs.authorId, followed)
-        }
+    fun followAuthorToggle(followed: Boolean) = viewModelScope.launch {
```

Comment:
Refactoring this with an expression body leaks the `Job` instance to the caller.
The implicit signature of the function becomes `fun followAuthorToggle(followed: Boolean) : Job`.

Reply:
That's right, I missed that. Thanks for mentioning it!